packages/@aws-cdk/aws-stepfunctions-tasks/lib/emr/emr-modify-instance-group-by-name.ts

```diff
@@ -73,38 +79,25 @@ export namespace EmrModifyInstanceGroupByName {
     /**
      * Specific list of instances to be protected when shrinking an instance group.
      *
-     * @default No instancesToProtect
+     * @default - None
      */
     readonly instancesToProtect?: string[];
 
     /**
      * Specific list of instances to be terminated when shrinking an instance group.
      *
-     * @default No instancesToTerminate
+     * @default - None
      */
     readonly instancesToTerminate?: string[];
 
     /**
      * Decommissioning timeout override for the specific list of instances to be terminated.
      *
-     * @default EMR selected default
+     * @default - EMR selected default
      */
```

Comment:
We usually specify the default directly here even if it's from a downstream.

Reply:
i'm punting on this change for now. 

rationale: it's not documented downstream. I will have to create a stack and use it to determine the behaviour.